src/components/views/ViewCampaign.jsx

```diff
@@ -127,18 +147,25 @@ const ViewCampaign = ({ match }) => {
       .catch(() => {
         setNotFound(true);
       });
-  }, []);
```

Comment:
If donationsTotal is changed everything is fetched from scratch! the better solution is to just update donations/leader board list of items.

Reply:
The Balances component needs it to be loaded again

`<Balances entity={campaign} />`